### DISCUSSION TITLE
What is the target date for Arcus Scripting 0.6.0?

### DISCUSSION THREAD

**Discussion by MassimoC** [arcus-azure/arcus.scripting#256] [Q&A]
Looking for the final fix for the #197 and there were still something wrong in the 0.5.0
Thanks
M

**Comment by stijnmoreels** [comment]
Yes, there was still something wrong with that fix, as it wrongly considered Windows systems. That was fixed in #244 .

**Reply by stijnmoreels** [reply]
We could create a fix release (v0.5.1) because the most recent changes are all fixes and there's still a lot of open features in v0.6 before we could release that one. @pim-simons , what do you think?

**Reply by pim-simons** [reply]
Go for it! 🚀

**Reply by pim-simons** [reply]
@MassimoC 0.5.1 has been released which contains the fix, see https://github.com/arcus-azure/arcus.scripting/releases/tag/v0.5.1

**Reply by stijnmoreels** [reply]
@MassimoC , does that answer your question and concern? We can mark as 'answered' otherwise.
